source/modules/MEI.critapp.xml

```diff
@@ -64,7 +64,8 @@
     </classes>
   </classSpec>
   <elementSpec ident="app" module="MEI.critapp">
-    <desc xml:lang="en">(apparatus) – Contains one or more alternative encodings.</desc>
+    <gloss versionDate="2023-06-16" xml:lang="en">apparatus entry</gloss>
```

Comment:
```suggestion
    <gloss versionDate="2023-06-16" xml:lang="en">apparatus</gloss>
```

Reply:
Took this from the original in TEI.

Reply:
@rettinghaus Can you resolve this? There seems to be a preference to go with just "apparatus" for now.